examples/rvc-app/README.md

```diff
@@ -0,0 +1,132 @@
+# RVC example app
+
+This example app is meant to demonstrate an implementation of a Matter Robotic
+Vacuum Cleaner device.
+
+## State machine
+
+Below is a diagram describing the state machine for this app.
+
+![state machine](RVC_app_state_diagram.png)
+
+This app can support most of the tests in the test plans.
+
+## Out-of-band messages
```

Comment:
Perhaps call these out-of-band events or change the name of the key from "Name" to "Event"

Reply:
I am using the `"Name"` key to keep it similar to the other apps. Including the all-clusters-app. Note, not all messages are events, some are sort of commands.